lib/utils/error-messages.js

```diff
@@ -284,6 +289,10 @@ function getPropertyRequiredErrorMessage(report, executionPlatformLabel) {
     return `${ getIndefiniteArticle(typeString) } <${ typeString }> with <Form type: Camunda forms> must have a defined <Form JSON configuration>`;
   }
 
+  if (is(node, 'bpmn:SequenceFlow') && requiredProperty === 'conditionExpression') {
+    return `${ getIndefiniteArticle(typeString) } <${ typeString }> must have a defined <Condition expression> or be the default <Sequence Flow>`;
```

Comment:
Is this the place where we workaround broken 8.1 behavior? (Default has to have a condition?) I'd suggest to unbuild it and rather let 8.1 fix this.

Reply:
No, this is not a workaround. The rule is implemented according to the correct behavior (cf. https://github.com/camunda/bpmnlint-plugin-camunda-compat/blob/main/test/camunda-cloud/inclusive-gateway.spec.js).

Reply:
With correct you refer to existing 8.1-alpha5 behavior I guess?

I was specifically asking because this is _NOT_ the correct BPMN behavior I believe.

Default flows must not have conditions according to the spec. See also https://github.com/camunda/zeebe/issues/10310.

Reply:
From the specification:

> The default Sequence Flow should not have a conditionExpression.
Any such Expression SHALL be ignored.

As far as I understand, it's not an error. We could still decide to show a lint error which we don't at the moment. That would require a change in [bpmnlint-plugin-camunda-compat](https://github.com/camunda/bpmnlint-plugin-camunda-compat/).

Reply:
Understood. Looking at the tests you provided we do support the standard BPMN semantics: https://github.com/camunda/bpmnlint-plugin-camunda-compat/blob/main/test/camunda-cloud/inclusive-gateway.spec.js#L47.

Thanks for sharing that!